### ISSUE TITLE
Please add https://rofr.in PWA

### ISSUE THREAD

**Issue by kanodia** [operasoftware/pwa-list#82]
Hello

Please add our web app called as Rofr.
Here is the link https://rofr.in

Made using polymer

**Comment by pepelsbey** [comment]
Your onboarding dialog suggest Chrome as the only supported browser. What current Opera and Firefox (and future Edge) users supposed to do?

**Comment by kanodia** [comment]
No, that is not true.

We support physical web, which was earlier part of chrome. If the first
screen where we have shown physical web tutorial is making you believe that
we support chrome only then I ll assure you that design will be changed in
a day or two.

Our pwa, Rofr.in is supported by all browsers including opera and firefox

**Comment by pepelsbey** [comment]
Please let us know when it’s changed so we could add it to our list, thank you.

**Comment by kanodia** [comment]
Hello Vadim

It has been changed. Please shift refresh https://Rofr.in  to view the
updated version

Thank you

Looking forward to see Rofr in your list of Pwa

Thanks again

**Comment by kanodia** [comment]
Hello Vadim

Did you get a chance to see the updated version of https://rofr.in

**Comment by andreasbovens** [comment]
@kanodia it seems like no native install banner is triggered... This is a condition for inclusion in pwa.rocks.

**Comment by andreasbovens** [comment]
Still the same problem... @kanodia any chance to fix this?

**Comment by kanodia** [comment]
No, We have web app manifest in our PWA and it does trigger native install
banner. Please check again
We have more than 1000 users who have Added Rofr to homescreen

*Abhishek Kanodia*
Under Graduate Student
BITS Pilani

**Comment by shwetank** [comment]
@kanodia It seems you don't have a 'short_name' field in your manifest.json file. This is required as mentioned in the readme. Please add that.
